lib/ansible/runner/credential/rhv_credential.rb

```diff
@@ -0,0 +1,49 @@
+module Ansible
+  class Runner
+    class RhvCredential < Credential
+      def self.auth_type
+        "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::RhvCredential"
+      end
+
+      # Modeled off of rhv injectors for awx:
+      #
+      #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/models/credential/__init__.py#L1035-L1054
+      #
+      def env_vars
+        write_credentials_file
+        {
+          "OVIRT_INI_PATH" => rhv_credentials_file,
+          "OVIRT_URL"      => auth.host || "",
+          "OVIRT_USERNAME" => auth.userid || "",
+          "OVIRT_PASSWORD" => auth.password || "",
+        }
+      end
+
+      def write_credentials_file
+        rhv_data = %W[
+          [ovirt]
+          ovirt_url=#{auth.host}
+          ovirt_username=#{auth.userid}
+          ovirt_password=#{auth.password}
+        ]
+
+        # TODO:  NickL:  Should we support this?
+        #
+        # awx ref:
+        #
+        #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/models/credential/__init__.py#L1046
+        #
+        # rhv_data << "ovirt_ca_file=#{auth.auth_key}" if auth.auth_key
```

Comment:
I would say no since it's not in the dialog.

Reply:
With most of these, I am mostly asking since AWX supports it, and didn't add it because we specifically don't support it yet.